azure-pipelines.yml

```diff
@@ -62,15 +62,6 @@ jobs:
     pool:
       vmImage: windows-2019
     steps:
-      - task: CmdLine@2
```

Comment:
removed some redundant steps as we already have a clean solution.
Also copy changes from MCT azure-pipeline: https://github.com/CommunityToolkit/Maui/pull/283/files